### DISCUSSION TITLE
[V1] How to pass multiple values to a same css prop?

### DISCUSSION THREAD

**Discussion by Calvein** [chakra-ui/chakra-ui#2297] [Q&A] [answered]
Hello, I'm trying to set multiple values to a same css prop, likewise: 

```css
main {
  min-height: 100vh;
  min-height: fill-available;
}
```

I'm pretty sure there are no way to do it in JSX so I'm trying to create my own element. In v0.8 I could use `styled` from emotion like that:

```js
styled.main {
  min-height: 100vh;
  min-height: fill-available;
}
```

But since it's not a peer-dep of chakra 1, I was wondering how I should handle it, if I should add it to my projet or if chakra added it (I did see `chakra.main` but it's not the same apparently) ?

Thanks 🤗

**Comment by ljosberinn** [comment, marked as answer]
We've had that question on [the discord](https://discord.com/channels/660863154703695893/695564068895064094/764159344731750440) in the past and here's the solution:
emotion supports this by passing a nested array to the value like this:
```js
height={[
  ['100vh', '-webkit-fill-available']
]}
```

https://cdn.discordapp.com/attachments/695564068895064094/764159342349778944/unknown.png

Re adding emotion: there is `import { chakra } from '@chakra-ui/core'` if you want to use chakra props on an unstyled element, e.g. `<chakra.section>...` or `<chakra.a>`, but it's exclusively the regular component API, no tagged template literals.

**Reply by Calvein** [reply]
Thanks @ljosberinn, it works perfectly 👍
(btw `fill-available` is handled by autoprefixer if it can help someone).

**Reply by chinanderm** [reply]
This syntax unfortunately no longer works in v1. How can it be accomplished now?

**Reply by ljosberinn** [reply]
Are you certain? I can't test it right now but this is a feature from @emotion/react, not chakra and I'd be surprised if it was removed there.

Edit: still allowed in emotion according to [their docs](https://emotion.sh/docs/object-styles#fallbacks)

**Reply by chinanderm** [reply]
@ljosberinn The Typescript typings don't allow it.

`<Box height={[['100vh', '-webkit-fill-available']]} />`

```
Type '[string, string]' is not assignable to type 'Union<number | (string & {}) | "px" | "xl" | "-moz-initial" | "inherit" | "initial" | "revert" | "unset" | "-moz-max-content" | "-moz-min-content" | "-webkit-fit-content" | "fit-content" | ... 54 more ... | "auto"> | null'.
  Type '[string, string]' is not assignable to type '"auto"'.ts(2322)
```

Am I misunderstanding where this syntax can be applied to?

**Comment by jhujhul** [comment]
I faced the same problem, and ended up using the `css` function from emotion (which should be added to your project according to chakraui [getting started page](https://chakra-ui.com/docs/getting-started#installation)).

````jsx
import { css } from '@emotion/react';

<Flex
    css={css`
      min-height: 100vh;
      min-height: -webkit-fill-available;
    `}
````

**Comment by terrierscript** [comment]
I found solve this with `sx` property.

```jsx
<Box
  sx={{
    minH: "100vh",
    "&":{
      minH: "-webkit-fill-available"
    }  
  }}
/>
```
